### PR TITLE
added online route

### PR DESCRIPTION
**Pull request by Oleksii30**

### CODE REVIEW

**Comment by undead404** [comment]
I accidentally pushed here, that doesn't matter @Oleksii30

File: src/routes/online-status/online-status.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import withStyles from 'isomorphic-style-loader/withStyles';
```
Comment (undead404) [inline src/routes/online-status/online-status.js]: Could you rewrite it using `useStyles`? Take other components as examples

**Comment by Oleksii30** [comment]
yes, sure

пт, 20 бер. 2020 о 10:29 Vitalii Perehonchuk <notifications@github.com>
пише:

File: package.json [hunk 2]
```diff
@@ -96,7 +96,7 @@
     "eslint-plugin-jest": "~23.8.2",
     "eslint-plugin-jsx-a11y": "~6.2.3",
     "eslint-plugin-lodash": "~6.0.0",
-    "eslint-plugin-postcss-modules": "~1.1.2",
+    "eslint-plugin-postcss-modules": "^1.1.2",
```
Comment (undead404) [inline package.json]: But why?

File: src/routes/index.js [hunk 3]
```diff
@@ -26,11 +26,12 @@ const routes = [
         load: () => import(/* webpackChunkName: 'dashboard' */ './dashboard'),
         path: '',
       },
+      // Wildcard routes, e.g. { path: '(.*)', ... } (must go last)
       {
-        load: () => import(/* webpackChunkName: 'login' */ './login'),
-        path: '/login',
+        load: () =>
+          import(/* webpackChunkName: 'online-status' */ './online-status'),
+        path: '/online-status',
```
Comment (undead404) [inline src/routes/index.js]: Please restore `/login` route created by Oleksandr.

**Pull request merged**